app/src/ui/history/commit-list-item.tsx

```diff
@@ -391,19 +391,19 @@ export class CommitListItem extends React.PureComponent<
           ? `Squash ${count} Commits…`
           : `Squash ${count} commits…`,
         action: this.onSquash,
+        enabled: !this.props.disableSquashing,
       })
     }
 
     return items
   }
 
   private canCherryPick(): boolean {
-    const { onCherryPick, isCherryPickInProgress } = this.props
+    const { onCherryPick, disableCherryPicking } = this.props
     return (
       onCherryPick !== undefined &&
       this.onSquash !== undefined &&
-      isCherryPickInProgress === false
-      // TODO: isSquashInProgress === false
```

Comment:
>TODO: isSquashInProgress === false 
Need to revisit this.. separate PR. I tested it and it is a bug.. but not as ugly as a bug as I expected.. the clicking a multi commit action while another is in progress (during a merge conflict) just doesn't work/do anything.